pkg/detectors/browshot/browshot.go

```diff
@@ -20,7 +21,7 @@ var (
 	client = common.SaneHttpClient()
 
 	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
-	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"browshot"}) + `\b([a-zA-Z-0-9]{28})\b`)
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"browshot"}) + `\b([a-zA-Z-0-9]{32})\b`)
```

Comment:
The token I received is 24 characters long. I couldn't find an option in the [Dashboard](https://browshot.com/dashboard#introduction) to refresh the token, so I created a second account for confirmation 😅. However, I received another API key that is also 24 characters long. I believe this needs further investigation to ensure our current regex is accurate.

Reply:
Weird, in my case it was 32, don't know the reason why exactly. They haven't even specified in the documentation.

Reply:
Any confirmation here @sahil9001?

Reply:
I did contact them, but they didn't reach back yet.